qemu/tests/qemu_guest_agent.py

```diff
@@ -1375,6 +1373,134 @@ def setup(self, test, params, env):
             args = [params.get("gagent_serial_type"), params.get("gagent_name")]
             self.gagent_create(params, self.vm, *args)
 
+    def _fswrite_check(self, flag):
+        """
+        Get the result from vm and check if the time stamp is not continuous
+        if not continuous,guest file system is frozen.
+        """
+        # init fs status to 'thaw'
+        fs_status = "thaw"
+        file_name = "/tmp/fsfreeze_%s.txt" % flag
+        process.system("rm -rf %s" % file_name)
+        self.vm.copy_files_from("C:\\fsfreeze.txt", file_name)
+        with open(file_name, 'r') as f:
+            list_time = f.readlines()
+
+        for i in list(range(0, len(list_time))):
+            list_time[i] = list_time[i].strip('')
+
+        for i in list(range(1, len(list_time))):
+            num_d = float(list_time[i]) - float(list_time[i - 1])
+            if num_d > 5:
```

Comment:
why use 5 to confirm FS status?

Reply:
@xiagao

Reply:
@xiangchunfu Fsfreeze is limited up to 10 seconds for windows guest. And from manually test, sometimes it's 9s, so change it to 8s.